### ISSUE TITLE
Collection item template functionality should be moved in a tab

### ISSUE THREAD

**Issue by atarix83** [DSpace/dspace-angular#1058]
Regarding the work on usability/accessiblity done in #728 it is considered more appropriate to move the functionality for creating/editing a collection's item template in a separate tab 
![Schermata da 2021-03-16 17-41-17](https://user-images.githubusercontent.com/2486489/111346950-07841f80-867f-11eb-8242-7f73551df995.png)

**Comment by tdonohue** [comment]
Agreed, this might be a good change to make. However, I'd like to have user feedback on this first, as the feature works as is.  Moving this to our 7.1 board to re-discuss during that release

**Comment by abelc-escire** [comment]
Hi @lgeggleston, I hope you are doing great. Could you please assign me the task?

**Comment by lgeggleston** [comment]
Hi @abelc-escire, you're assigned - however, with a caveat: I would say all of the comment from [this similar ticket](https://github.com/DSpace/dspace-angular/issues/1059#issuecomment-5295751221) apply in this case.  You may want to confirm there is still interest before proceeding; See Tim's comment above as well.  Thanks!

**Comment by abelc-escire** [comment]
Hi @lgeggleston, thanks for following up. I’ve verified the status, and the change has not been applied yet. I’ll go ahead and apply it so it’s ready in case it’s needed in the future.

Regards!
